internal/driver/driver.go

```diff
@@ -134,22 +141,24 @@ func (d *Driver) Initialize(sdk interfaces.DeviceServiceSDK) error {
 	}
 
 	// check to see if rtsp-simple-server file/binary exists
-	_, err = os.Stat(RtspServerCmd)
+	rtspExecutable := d.ds.DriverConfigs()[RtspServerExe]
```

Comment:
Should handle not set in a better way, like check for empty and return error that it isn't set